keras/backend/tensorflow_backend.py

```diff
@@ -1866,6 +1866,26 @@ def batch_normalization(x, mean, var, beta, gamma, epsilon=1e-3):
     # Returns
         A tensor.
     """
+    if ndim(x) == 4:
+        # The CPU implementation of FusedBatchNorm only support NHWC
```

Comment:
NCHW should be supported on GPU since it offers better performance. For conv ops we have a test checking whether we run on GPU or not, check it out

Reply:
Agree